### PR TITLE
Team-4 Updates

### PR DESCRIPTION
**Pull request by jiggylepcha**
-Column names to ChargePolicy has been added

### CODE REVIEW

File: src/main/java/com/nucleus/chargepolicy/service/ChargePolicyServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+@Service
+public class ChargePolicyServiceImpl implements ChargePolicyService {
+
+    @Autowired
+    private ChargePolicyDao chargePolicyDao;
+    public ChargePolicyDao getChargePolicyDao() {
+        return chargePolicyDao;
+    }
+
+    public void setChargePolicyDao(ChargePolicyDao chargePolicyDao) {
```
Comment (Cactusmachete) [inline src/main/java/com/nucleus/chargepolicy/service/ChargePolicyServiceImpl.java]: why have a setter here if the Dao is autowired

File: src/main/java/com/nucleus/chargepolicy/service/ChargePolicyServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+   /* public void getChargePolicyList(ChargePolicy chargePolicy) {
+        System.out.println("In Service");
+        this.chargePolicyDao.getChargePolicyList();
+    }*/
+
+
+
+    public void setEligibilityPolicyDAO(ChargePolicyDao chargePolicyDao) {
```
Comment (Cactusmachete) [inline src/main/java/com/nucleus/chargepolicy/service/ChargePolicyServiceImpl.java]: same here with the autowired attribute

File: src/main/java/com/nucleus/chargepolicy/service/ChargePolicyServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        this.chargePolicyDao = chargePolicyDao;
+    }
+    public void insert(ChargePolicy chargePolicy) {
+        System.out.println("In Service");
+        this.chargePolicyDao.insert(chargePolicy);
+    }
+    public List<String> getChargeCodes(){
```
Comment (Cactusmachete) [inline src/main/java/com/nucleus/chargepolicy/service/ChargePolicyServiceImpl.java:37]: why is this method there, if its not getting anything from the Dao? is it used anywhere else?

**Review by Cactusmachete — CHANGES_REQUESTED** [review]

File: src/main/java/com/nucleus/chargepolicy/service/ChargePolicyServiceImpl.java (hunk 3 above)
Comment (jiggylepcha) [reply src/main/java/com/nucleus/chargepolicy/service/ChargePolicyServiceImpl.java:37]: I'll leave this here incase there is some dependency of this somewhere

**Pull request merged**